modules/installation-creating-aws-control-plane.adoc

```diff
@@ -151,9 +151,9 @@ directory. This value is the long string with the format
 the CloudFormation template for the security group and roles.
 <21> The type of AWS instance to use for the control plane machines based on your selected architecture.
 <22> The instance type value corresponds to the minimum resource requirements for
-control plane machines. For example `m6i.xlarge` is a type for `x86_64`
+control plane machines. For example `m6i.xlarge` is a type for AMD64.
 ifndef::openshift-origin[]
-and `m6g.xlarge` is a type for `arm64`.
+and `m6g.xlarge` is a type for ARM64.
```

Comment:
Previously, ARM64 was in backticks. Just commenting this in case they still need to be in backticks, but maybe you got different guidance for this update. Applies also to line 109.